po/gl.po

```diff
@@ -325,7 +325,8 @@ msgstr[1] "{{ count }} autoras en este blog: "
 
 msgid "Login or use your Fediverse account to interact with this article"
 msgstr ""
-"Conéctese ou utilice a súa conta no fediverso para interactuar con este artigo"
+"Conéctese ou utilice a súa conta no fediverso para interactuar con este "
+"artigo"
```

Comment:
why does this file keep changing with every new build?

Reply:
it is updated every time you run Plume. However, extracting i18n related task to an external tool may be better… :thinking: